.github/pull_request_template.md

```diff
@@ -1,16 +1,16 @@
-<!-- Please don't forget your JIRA link -->
-**JIRA:**
+<!-- Please don't forget your Issue link -->
+close/fixes #NUMBER
```

Comment:
I would just update this according to this https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
```suggestion
**Closes/Fixes/Resolves** #ISSUE-NUMBER
```